src/runner/src/bin/main.rs

```diff
@@ -1,10 +1,47 @@
 use structopt::StructOpt;
-use fluvio_future::task::run_block_on;
 use fluvio_run::RunCmd;
 
+use tracing::Level;
+use tracing_subscriber::{Registry, prelude::*};
+use tracing_subscriber::filter::Directive;
+
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    fluvio_future::subscriber::init_tracer(None);
```

Comment:
make jaeger optional.  if `telemetry` feature is not defined then default to previous behavior.